content/departments/engineering/dev/process/escalation-engineer-rotation.md

```diff
@@ -42,6 +42,16 @@ This is a new program, so any and all of these points are flexible, but here is
 
 If you have questions or want to participate, please post in #escalation-engineering.
 
+## Tips for new escalation engineers
+
+- Ask around, nobody is expecting that you come up with the answer or context entirely on your own.
+- Asking what happens if the request is not addressed is a good way of getting to the importance of an ask. It will help you to delegate the task if you need to, because you'll be able to convey that context to the new owners.
+- When you do, precise that you're wearing the esc-eng hat, and link back to the original ask, this will help others to understand that what you're asking is important
```

Comment:
```suggestion
- When you do (since you're wearing the esc-eng hat) link back to the original ask, this will help others to understand that what you're asking is important.
```

Reply:
```suggestion
When you do, specify that you're wearing the esc-eng hat, and link back to the original request. This will help others to understand that what you're asking is important.
```